apps/checkout/package.json

```
@@ -47,5 +47,5 @@
     "node": ">=16.0.0",
     "pnpm": "^7.5.0"
```

We should update it just like in main `package.json`:

```suggestion
    "pnpm": ">=7.7.1"
```